impeller/entity/shaders/vertices.frag

```diff
@@ -4,10 +4,15 @@
 
 #include <impeller/types.glsl>
 
+uniform FragInfo {
+  float alpha;
```

Comment:
Might make sense to tack this onto `position_color.vert` instead and multiply the alpha in there.

Reply:
Let me revist this when I do DrawVertices next, since I need to adjust the contents there anyway.